jira_agile_metrics/config_test.py

```diff
@@ -450,8 +445,7 @@ def test_config_to_options_extends():
         - 0.2
 
     Backlog column: Backlog
-    Committed column: Committed
-    Final column: Test
```

Comment:
Again - changes a test that would have passed previously.

Reply:
This didn't make sense before:
```
Workflow:
    Backlog: Backlog
    In progress: Build
    Done: Done
```
and
```
    Backlog column: Backlog
    Committed column: Committed
    Final column: Test
    Done column: Done
```
This is wrong in two ways:
1. There is no workflow stage called `Committed`. When setting `Committed column: Committed` the sanity check should have thrown an exception.
2. As for me, the assumption in the assertion is wrong.

With the changes and new sanity checks in `config.py`, this config will trigger an exception.